pyrefinebio/token.py

```diff
@@ -93,19 +103,15 @@ def save_token(self):
                 "Please create a new token."
             )
 
-        config = Config()
-        config.save("token", self.id)
+        config.token = self.id
+        config.save()
 
 
     @classmethod
     def get_token(cls):
-        """gets the token that's currently saved to the config file.
-
-        The default config file is ~/.refinebio.yaml, but
-        you can use the environment variable `CONFIG_FILE` to change this path
-        """
-        config = Config()
+        """Gets the token that's currently set to Config."""
```

Comment:
I think `load_token` might be more intuitive here because `get_token` might make people think it's getting it from the API.